doc/cookbook/source/conf.py

```diff
@@ -303,6 +303,7 @@
 
 # options for external link plugin
 extlinks = {
+'cookbook_classifier': ('http://shogun.ml/cookbook/latest/examples/classifier/%s.html',        ''),
```

Comment:
I like this idea, but please use relative links? I.e. remove the `http://shogun.ml/cookbook/latest/`